docs/introduction.md

```diff
@@ -4,145 +4,62 @@
-
-Firstly, application developers have to understand quite a few new 
-concepts and resource specifications before deploying their applications.
-Sometimes, this is not an easy task. For example, it may take a while
-for developers to understand some runtime related configurations
-such as `allowPrivilegeEscalation` which they rarely touch in their daily work.
-The pervasive use of CRDs makes the situation even worse since developers
-have to learn all the CRD schemas as well. 
-
-Moreover, since developers have to specify the details of all the
-involved resource objects, application management in Kubernetes soon
-becomes a headache of handling a large amount of resource YAML files.
-We have seen that the lack of API abstraction had led to low productivity,
-unexpected errors or misconfigurations in production. Application developers
-start to question "why am I bothered with these many YAML files?"
-
-On the other hand, Kubernetes is a platform for platforms. Literally, the extended
-capabilities provided by the platform builders are the keys to support the
-Kubenetes ecosystems. Nowadays, a typical production Kubernetes would install
-dozens of customized operators and plugins. Since abstracting the Kubernetes
-APIs is a highly opinionated process, the resultant simplified interfaces
-would only make sense had the decision makers been the platform builders.
-Unfortunately, the platform builders today face the following dilemmas when
-dealing with the API abstraction:
-
-- There is no tool or framework for them to easily extend the API abstraction
-  if any. They have to rely on existing interfaces or assumptions for new
-  capabilities which may be only suitable for specific user cases.
-  This makes extending such platform to leverage broader Kubernetes ecosystems
-  almost impossible.
-
-- It could be painful to develop new capability based on existing interfaces.
-  The operator design could easily becomes a marathon negotiation in order
-  to meet the abstraction requirement.
-
-In the end, application developers may complain that Kubernetes is hard to use
-and the platform builders may want to help but they cannot do it easily.
+The modern trend of cloud-native technology is moving towards pursuing consistent application delivery across clouds and on-premises infrastructures using Kubernetes as the common abstraction layer. Kubernetes, although excellent in abstracting away low-level infrastructure details, does introduce another layer of complexity to application developers, namely: pods, port exposing, privilege escalation, resource claims, CRD, and so on. We’ve seen such a learning curve and the lack of developer-facing abstraction have impacted user experience, slowed down productivity, led to unexpected errors or misconfigurations in production. People start questioning the value of this revolution: "why am I bothered with that many YAML files?" they asked.
```

Comment:
- abstracting away low-level infrastructure details. (remove "away", the gramma should be abstract sth away from sth);

- namely -> namely understanding the concepts of 
- such a learning curve-> such a nontrivial learning curve;
- Remove they asked in the end of this paragraph.